drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1,4 +1,4 @@
-function DragNDropTemplates(url_name) {
+function DragNDropTemplates(configuration) {
```

Comment:
@mtyaka Since you're touching this line, could you take the opportunity to make the spelling consistent with the remaining code (i.e., change `DragNDropTemplates` to `DragAndDropTemplates`)? That would be great :) There's only two occurrences of this spelling in the entire repo, and both of them are in this file, so it should be a quick change.